lib/modules/platform/types.ts

```diff
@@ -147,6 +147,11 @@ export type EnsureCommentRemovalConfig =
 
 export type EnsureIssueResult = 'updated' | 'created';
 
+export interface RepoCacheConfig {
+  commit: string;
+  blob: string;
+}
```

Comment:
For some API blob SHA can be used, for another we have to tell commit SHA instead